style.css

```diff
@@ -0,0 +1,314 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  height: 100vh;
+  padding: 0 50px;
+}
+
+.grid-container {
+  display: grid;
+  grid-template: repeat(6, minmax(150px, 1fr)) / repeat(6, minmax(150px, 1fr));
+  grid-template-areas: "sidebar header header header header header" "sidebar content content content content content" "sidebar content  content content content content" "sidebar content content content content content " "sidebar content content content content content ";
```

Comment:
Great work providing detail properties for each area. It would be nice great indent the values something like
```
grid-template-areas: "sidebar header ... ... ..."
                     "sidebar content ... ... ..."
                     "..................";
 ```
what do you think?